input/outfit-acquisition/gem.json

```diff
@@ -7,6 +7,8 @@
     9,16,15,58,20,31,6,44,10,53,1,21,23,51,77,5,19,7,72,56,43,54,38,25,46,55,17,75,37,14,49,28,30,2,22,45,24,42,34,13,50,3,76,11,47,39,8,32,48,41,78,40,26,36,74,33,4,79,80,
     81,82,83,84,85,86,88,91,92,93,94,97,98,99,
     100,102,103,104,105,106,107,108,109,
-    110,111,112,114,115,116,117,118,119
+    110,111,112,114,115,116,117,118,119,120,
+    // Standalone - Clothing - Emblem
+    95
```

Comment:
Might be better to remove the acquisition method entirely. The only time it was available it was a free item, so not technically a gem item.

Reply:
There's quite a few of "once in gem store, never back" collectibles, not really know what to do with them, might return some day for all we know (few of them have). I'd keep them for now, if just for historical purposes, as we do with achievements and whatnot.

If we ever wind up adding some kind of "Unavailable" category, we could move all of those in there. Maybe automatically categorize any entries with no categories (also those with only the Guaranteed Wardrobe Unlock category) into said unavailable category, so there's no orphaned items at any point whatsoever (except for new releases, which would never be classified until they're manually included).